packages/perseus/src/widgets/interactive-graphs/backgrounds/axis-ticks.tsx

```diff
@@ -48,7 +48,7 @@ const YGridTick = ({y, range}: {y: number; range: [Interval, Interval]}) => {
     const showLabel = shouldShowLabel(y, range);
 
     return (
-        <g className="tick">
+        <g className="tick" aria-hidden={true}>
```

Comment:
Why hide this content from screen readers? 🤔

Reply:
It makes you go through all the axis tick labels, so it sounds like "1, 2, 3, 4, 5, 6, 7, 8, 9, 10, -1, -2, -3, -4, -5..." with no context for why it's just randomly saying numbers.